.github/workflows/release.yml

```diff
@@ -0,0 +1,27 @@
+name: Release
+
+on:
+  release:
```

Comment:
Will this create 2 deployment markers if we both create a GitHub release and merge the PR to `main`?

Reply:
Yeah. I wasn't sure if there was a scenario where we would do one without the other (such as publish a release from an _older_ commit). It's a bit of a weird use case, I could just remove the `release` part since we have no tooling set up for that right now anyway.

Reply:
Awesome. Definitely something we can add later if need be